authors/apps/article/serializers.py

```diff
@@ -0,0 +1,84 @@
+'''Serializers allow complex data
+such as querysets and model instances
+ to be converted to
+native Python datatypes that can then
+be easily rendered into JSON, XML or other content types.'''
+
+from rest_framework import serializers
+from django.apps import apps
+from .models import RateArticle
+from authors.apps.profiles.serializers import ProfileListSerializer
+
+
+TABLE = apps.get_model('article', 'Article')
+Profile = apps.get_model('profiles', 'UserProfile')
+
+NAMESPACE = 'article'
+fields = ('id', 'slug', 'image', 'title', 'description', 'body', 'user',)
+
+
+class ArticleSerializer(serializers.ModelSerializer):
+    update_url = serializers.HyperlinkedIdentityField(
+        view_name=NAMESPACE + ':update', lookup_field='slug')
+    delete_url = serializers.HyperlinkedIdentityField(
+        view_name=NAMESPACE + ':delete', lookup_field='slug')
+    author = serializers.SerializerMethodField(read_only=True)
+
+    class Meta:
+        model = TABLE
+
+        fields = fields + ('author', 'update_url', 'delete_url')
+
+    def get_author(self, obj):
+        try:
+            serializer = ProfileListSerializer(
+                instance=Profile.objects.get(user=obj.user)
+            )
+            return serializer.data
+        except BaseException:
+            return {}
+
+    def update(self, instance, validated_data):
```

Comment:
Maybe you should try to provide some Docstrings  in the future to describe what each  method is doing

Reply:
I had heavy comments all over my work which brought alot of linting issues thus i decided to scrap them all of and left only a-few. I will however explain everything during code review.